src/client/src/commons/form/profile/components/profileLayers/profileLayers.js

```diff
@@ -68,6 +74,65 @@ const ProfileLayers = props => {
     },
   );
 
+  const deleteLithologicalDescriptionMutation = useMutation(
+    async id => {
+      const result = await deleteLithologicalDescription(id);
+      return result;
```

Comment:
PP: Die Variable `result` kann wegoptimiert werden. Dito für alle anderen Fälle weiter unten.
```suggestion
      return await deleteLithologicalDescription(id);
```